components/ecoindex/scraper/scrap.py

```diff
@@ -58,11 +59,14 @@ async def get_page_analysis(self) -> Result:
 
     async def scrap_page(self) -> PageMetrics:
         async with async_playwright() as p:
-            browser = await p.chromium.launch()
+            browser = await p.chromium.launch(headless=False)
+            user_agent = await get_user_agent()
+            print(f"Make request to {self.url} with {user_agent}")
```

Comment:
Replace the print statement with a logging statement for better debugging and production readiness. [important]